docs/rules/accessibility/image-has-alt.md

```diff
@@ -0,0 +1,25 @@
+# Image Has Alt
+
+## Rule Details
+
+`<img>` should have an alt prop with meaningful text or an empty string for decorative images.
```

Comment:
Shouldn't be `must` instead of `should`?

Reply:
`must` does come off a bit stronger but I think both are ok :)